servers/physics_2d/shape_2d_sw.cpp

```diff
@@ -691,6 +691,9 @@ bool ConcavePolygonShape2DSW::contains_point(const Vector2 &p_point) const {
 }
 
 bool ConcavePolygonShape2DSW::intersect_segment(const Vector2 &p_begin, const Vector2 &p_end, Vector2 &r_point, Vector2 &r_normal) const {
+	if (segments.size() == 0 || points.size() == 0)
+		return false;
```

Comment:
Why not raising an error as in the other cases?

Reply:
In the other cases, we're failing and returning a default value. In this case we can legitimately say there won't be an intersection if there are no segments or points.

Reply:
Alright, makes sense. It just needs brackets because of the coding style on master.

Reply:
Brackets added.